tests/test_temporal_model_mixin.py

```diff
@@ -223,11 +223,11 @@ def test_clock_tick_editing(self, session, newstylemodel):
         session.commit()
 
         activity = models.Activity(description="Activity Description #2")
-        with newstylemodel.clock_tick(activity=activity):
-            newstylemodel.description = "this is new"
-            newstylemodel.int_prop = 2
-            newstylemodel.bool_prop = False
-            newstylemodel.datetime_prop = datetime.datetime(2017, 2, 10)
+        newstylemodel.activity = activity
+        newstylemodel.description = "this is new"
+        newstylemodel.int_prop = 2
+        newstylemodel.bool_prop = False
+        newstylemodel.datetime_prop = datetime.datetime(2017, 2, 10)
```

Comment:
Do you have a tests that:
- not providing an activity raises an error
- activity and tick are cleared after save and rollback